src/main/java/jimmy/mcgymmy/model/food/Macronutrient.java

```diff
@@ -46,9 +46,7 @@ private boolean isValidAmount(int amount) {
         return amount >= 0 && amount < 1000;
     }
 
-    private String getMessageConstraint() {
-        return this.getMacronutrientType() + " amount can only take in value larger than 0 and less than 1000";
-    }
```

Comment:
I think we should move `"amount can only ..."` outside to be a string called `MESSAGE_CONSTRAINT_SUFFIX`. The old implementation works just as fine as the new one. The old one even has fewer repeated code. So why do we need to change?

Reply:
It is to make sure all of them point to the same message so that the messages shown will be consistent everywhere

Reply:
The old one also makes sure all of them point to the same message, isn't it